Sources/BCCommon/Commons.swift

```diff
@@ -10,6 +10,17 @@ public struct BCDashPattern {
   }
 }
 
+public struct BCCubicCurve {
+  public let control1: CGPoint
+  public let control2: CGPoint
+  public let to: CGPoint
+  public init(_ control1: CGPoint, _ control2: CGPoint, _ to: CGPoint) {
```

Comment:
Explicit arguments names?

Reply:
rn it is consistent with all other types declared in Commons.swift.

Reply:
And this has already brought us bad luck. That's a sign to be more explicit.